tasks/adduser.yml

```diff
@@ -11,8 +11,9 @@
   tags:
     - adduser
     - users
+  when: arh_modify_adduser_conf is true
```

Comment:
`is true` isn't needed

Reply:
Agreed. Much cleaner without the "is true".